.github/workflows/build.yml

```diff
@@ -112,6 +112,11 @@ jobs:
         uses: step-security/harden-runner@002fdce3c6a235733a90a27c80493a3241e56863 # v2.12.1
         with:
           egress-policy: audit
+          allowed-endpoints: >
+            8.8.8.8:53
```

Comment:
what is this for?

Reply:
I thought it might help to resolve the DNS resolution issue inside the podman containers, but it hasn't worked so far

Reply:
I've removed this now